lib/vector-common/src/internal_event/mod.rs

```diff
@@ -104,6 +106,10 @@ pub struct Count(pub usize);
 #[derive(Clone, Copy)]
 pub struct CountByteSize(pub usize, pub usize);
 
+// Wrapper types used to hold inputs for registering an event
+
+pub struct Output(pub Option<SharedString>);
```

Comment:
It is probably just my lack of understanding but thought it was odd to see `Output` below a comment talking about inputs.

Reply:
The registered event uses a source or transform output name as a parameter for registration. I'll reword to clarify.